### PR TITLE
add voetbal-nl

### PR DESCRIPTION
**Pull request by richardbeumer**
#### Pre-flight Checklist

Please ensure you've completed all of the following.

- [X] I have read the [Contributing Guidelines](https://github.com/ferdium/ferdium-app/blob/develop/CONTRIBUTING.md) for this project.
- [X] I agree to follow the [Code of Conduct](https://github.com/ferdium/ferdium-app/blob/develop/CODE_OF_CONDUCT.md) that this project adheres to.
- [X] I updated the version package [Updating](https://github.com/ferdium/ferdium-recipes/blob/main/docs/updating.md#4-updating-the-version-number) 

#### Description of Change

Add voetbal-nl recipe

### CODE REVIEW

**Comment by SpecialAro** [comment]
Hello 👋 thank you for taking the time to contribute!

May I ask you to remove all node updates (pnpm and dockerfile) from this PR, as we have a method for updating both ferdium-app and ferdium-recipes together?

Thank you!

**Review by vraravam — CHANGES_REQUESTED** [review]
I agree with @SpecialAro  - please remove the node and pnpm upgrades from this PR

**Comment by richardbeumer** [comment]
I'm sorry, I wasn't aware there was a method to keep the recipes in sync with the app. Node and docker updates are removed from this PR.

File: .gitignore [hunk 1]
```diff
@@ -11,3 +11,4 @@ server*.log
 *.bmp
 *.zwc*
 /all.json
+.pnpm-store
```
Comment (vraravam) [inline .gitignore:14]: can we remove this change as well?

**Review by vraravam — CHANGES_REQUESTED** [review]
Please also remove the `package-lock.json` file.

File: .gitignore (hunk 1 above)
Comment (richardbeumer) [reply .gitignore]: removed!!

**Review by vraravam — CHANGES_REQUESTED** [review]
Please revert changes in the `.gitignore` file. If needed, please squash your commits into a single one to ensure that the final changes are limited to the `recipesvoetbal-nl` folder alone

**Comment by richardbeumer** [comment]
squashed all commits to one for now.

**Review by vraravam — APPROVED** [review]

**Comment by vraravam** [comment]
@all-contributors please add @richardbeumer for code

**Pull request merged**